lib/jmap/mail/extensions/public_asset/public_asset.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:equatable/equatable.dart';
+import 'package:jmap_dart_client/http/converter/id_converter.dart';
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:jmap_dart_client/http/converter/identities/public_asset_identities_converter.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'public_asset.g.dart';
+
+typedef PublicAssetIdentities = Map<IdentityId, bool>;
```

Comment:
please test for this, also the failed cases in response body

Reply:
Done